app/controllers/evidence_controller.rb

```diff
@@ -10,6 +10,7 @@ class EvidenceController < NestedNodeResourceController
   before_action :initialize_nodes_sidebar, only: [ :edit, :new, :show ]
   skip_before_action :find_or_initialize_node, only: [:create_multiple]
   before_action :set_auto_save_key, only: [:new, :create, :edit, :update]
+  before_action :cancel_path, only: [:new, :create, :edit, :update]
```

Comment:
Sorry dudes I'm going to disapprove. We're adding an entire controller method, with a before_action- to generate a _link_.

At most this is essentially a URL helper, which may qualify is a view helper, but does not require bulking out controller functionality.